### DISCUSSION TITLE
Unable to fetch reactions

### DISCUSSION THREAD

**Discussion by vrocx28** [kevinzg/facebook-scraper#945] [Q&A] [answered]
` for post_res in get_posts(page_id, pages=10, cookies=cookies, options={"reactions": True}):`
`    post_json.append(post_res) `

in the list of json "reactions": null 
reply with a fix 
thanks

**Comment by vvendigo** [comment]
options={"comments": True, "reactions": True} works for me.
But you need also a valid session maybe.

**Reply by vrocx28** [reply]
i pulled the latest code 4-5 days ago, facing this problem ever since, thanks for the reply

**Comment by vrocx28** [comment, marked as answer]
the problem was in extractors.py line no. 214
if reactions["reactors"] and self.options.get("reactors") != "generator":
![problem](https://user-images.githubusercontent.com/18879049/206996930-d18c897d-e67d-4f1a-a206-b8cd9e4801b9.jpg)

changed it with
if reactions.get("reactors") and self.options.get("reactors") != "generator":
![solution](https://user-images.githubusercontent.com/18879049/206996981-961daad5-ad24-4323-80c6-e4dbe8340c34.jpg)
works fine now
